### DISCUSSION TITLE
Organizationsの500エラー

### DISCUSSION THREAD

**Discussion by Brutus03** [increments/qiita-discussions#480] [ヘルプ]
[インフラ勉強会](https://qiita.com/organizations/infra-workshop)のOrganizationsにアクセスすると、500エラーが返ってくるため、アクセスすることができません。

ご確認お願いします。

**Comment by tanaka-asami415** [comment]
@Brutus03 
ご迷惑をおかけし、申し訳ございません🙇
修正が完了し、ページも表示されるようになりましたので、ご確認いただけますと幸いです。

**Reply by Brutus03** [reply]
早速のご対応ありがとうございます。
ページの表示について確認しました。

ちなみに、何が原因だったのでしょうか？

**Reply by tanaka-asami415** [reply]
@Brutus03 
シンプルにバグだっため、再発防止に努めます🙇💦

**Reply by Brutus03** [reply]
承知しました。
本件についてはクローズで大丈夫です。
